frontend/app/src/views/settings/GeneralSettings.vue

```diff
@@ -138,6 +138,29 @@
               :error-messages="settingsMessages['dateDisplayFormat'].error"
               @change="onDateDisplayFormatChange($event)"
             ></v-text-field>
+
+            <v-text-field
+              v-model="amountDisplayFormat"
+              class="general-settings__fields__amount-display-format"
+              label="Amount display format (%T = thousands; %U = units; %D = decimals; %C = currency)"
+              type="text"
+              :messages="
+                'Resulting: ' +
+                (isValidAmountDisplayFormat(amountDisplayFormat)
+                  ? $options.filters.formatPrice(
```

Comment:
Would it be possible to move this check/calculation in a computed property?

Reply:
Is this what you were thinking about? https://github.com/rotki/rotki/pull/1269/files#diff-fe784788b0a861bf65d37abcf3e8ec8dR239-R251

Reply:
It's now using directly the AmountDisplay.vue component here